src/lib/button-toggle/OVERVIEW.md

```diff
@@ -0,0 +1,21 @@
+Buttons toggles aregroups of buttons that can be toggled on and off. They can be configured to 
+behave as checkboxes or as radio buttons. Button toggles are always part of a 
+`md-button-toggle-group`.
+
+<!-- example(button-toggle-overview) -->
+
+### Exclusive selection vs. multiple selection
+By default, `md-button-toggle-group` acts like a radio-button group- only one item can be selected.
```

Comment:
I wanted to refer to the general idea of a radio-button group vs. `md-radio-group` specifically.